perfaware/sim86/shared/Sim8086.cpp

```diff
@@ -7,7 +7,7 @@
 #pragma comment (lib, "sim86_shared_debug.lib")
 #define ArrayCount(Array) (sizeof(Array) / sizeof((Array)[0]))
 
-constexpr int RegisterNumber = 15;
+static constexpr int RegisterNumber = 15;
```

Comment:
You can make all the other global stuff static too, as long as they're only used in this file. So Registers, RegisterNames, SimulateInstructions.